2023/Nouman Wali/READEME.md

```diff
@@ -0,0 +1,108 @@
+- [ ] Contribute a solution to the issue you selected based on guidance in section "6. Contributing a solution".
+- [ ] TBC – Additional tasks specific to the Outreachy application process.
+
+### Checklist: Wagtail.org website accessibility
+
+🚧 This checklist is for candidates interested in working on the [Wagtail.org website accessibility](https://github.com/wagtail/outreachy/blob/main/project-ideas.md#wagtailorg-website-accessibility) project.
+
+- [ ] Write a short blog post which demonstrates your understanding of **accessibility**. In English, with at least one image, less than 500 words. Demonstrate your understanding _based on past experiences_. For example, any specific accessibility improvements you’ve done in the past, or specific issues you identified. The more specific the better. The post has to be posted on a publicly-available platform (Dev.to, Hashnode, Medium, your own website), and you must also add it as a new section in your personal file in this repository (so we can provide feedback on the contents).
+- [ ] Write a short blog post which demonstrates your understanding of **Django**, in less than 500 words. This can be an intro to an existing project of yours, or perhaps explanation of a specific code snippet you’re particularly proud of, or explanation of a specific problem you worked on. The more specific to your own experience the better. The post has to be posted on a publicly-available platform (Dev.to, Hashnode, Medium, your own website), and you must also add it as a new section in your personal file in this repository (so we can provide feedback on the contents).
+- [ ] Go through Wagtail’s tutorial: [Your first Wagtail site](https://docs.wagtail.org/en/stable/getting_started/tutorial.html). If you get stuck, ask for help on Slack or GitHub Discussions.
+- [ ] Set up [wagtail.org in Gitpod](https://github.com/wagtail/wagtail.org), via the "Open in Gitpod" link in the README.
+- [ ] Find an accessibility issue on the live [wagtail.org](https://wagtail.org/) website, and report it on the project’s issue tracker, with appropriate details (title, description, screenshot if applicable, why you think this is an accessibility issue).
+- [ ] Fork the [wagtail.org repository](https://github.com/wagtail/wagtail.org) and set up your own copy for development, either in Gitpod or on your own computer.
+- [ ] Contribute a solution to the issue you selected, by reproducing it on your copy of the project, and submitting a pull request to the main project.
+- [ ] TBC – Additional tasks specific to the Outreachy application process.
+
+## Research
+- [YouTube video for Wagtail tutorial](https://www.youtube.com/watch?v=SEPBP0DuoWg)
+- [Wagtail 2.9 documentation](https://docs.wagtail.io/en/v2.9/editor_manual/intro.html)
+- [A blog on wagtail CMS](https://blog.nonstopio.com/wagtail-an-open-source-cms-cec6b93706da)
```

Comment:
This is an article from 5 years ago. I’d really recommending making an effort to check the age of the resources you pick.

Reply:
Okay sure This is valuable advice. Thank you